src/it/nrepl-test/pom.xml

```diff
@@ -28,13 +28,15 @@
                 <artifactId>clojure-maven-plugin</artifactId>
                 <version>@project.version@</version>
                 <extensions>true</extensions>
-                <configuration>
-                    <port>4007</port>
+		    <configuration>
+	                <temporaryOutputDirectory>true</temporaryOutputDirectory>
+		      <port>4007</port>
                     <replScript>nrepl-script.clj</replScript>
                     <nreplMiddlewares>
-                        <middleware>lighttable.nrepl.handler/lighttable-ops</middleware>
+                        
 <!--
-                        <middleware>cider.nrepl.middleware.apropos/wrap-apropos</middleware>
+                        <middleware>lighttable.nrepl.handler/lighttable-ops</middleware>
```

Comment:
Does this still work? If yes, wow, I must definitely try light table again.

Reply:
In my tests, the lighttable middleware doesn't work with the latest nREPL. I changed the op code in the test from a lighttable specific one to a built-in one.

Reply:
Is lighttable still a thing?

Reply:
Not really sure. For me it has never been, but I heard something about the project resurrecting.